scripts/consumer-helper.sh

```diff
@@ -52,6 +52,7 @@ function copy_to_target {
     targetDir=$(resolve "$ROOT/../$projectDirectoryOrWorkspace/node_modules/@shopify/$package")
   else
     targetDir="$shopifyNodeModulesDir/$package"
+    echo "🌀 spin instance: `spin show | grep Shopify/$projectName | awk '{print $1}'` "
```

Comment:
Instead of adding a new message here, can we add to line 27 instead? This is where it's saying whether we're running spin or not